src/malloy/runtime.py

```diff
@@ -99,7 +101,9 @@ def load_source(self, source: str, import_path: str = None):
     self._log.debug("  file_name: %s", self._file_name)
     return self
 
-  async def get_sql(self, named_query: str = None, query: str = None):
+  async def compile_and_render(self,
```

Comment:
I think we should keep allowing the separate actions:
- get_sql
- run (get data)
- render (get html)

Reply:
Agree. This is a TODO for me. I want to introduce COMPILE_ONLY and COMPILE_AND_RENDER flows between malloy-py and malloy-service. So I think new functions should be:
1. compile -> returns sql
2. run -> no change
3. compile_and_render OR compile_run_render OR a better name -> returns sql result and html render content.